### PR TITLE
feat: mobile friendly NoopLayout

### PR DESCRIPTION
**Pull request by jessicamcinchak**
eg About page! tackling this task one layout at a time

tailwind responsive modifiers feel a bit fiddly, but getting the hang of it i think

this was a useful section of the tailwind docs! https://tailwindcss.com/docs/responsive-design#targeting-mobile-screens

Screenshot from 2023-01-25 19-22-30

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ const AboutPage = () => {
           </blockquote>
         </div>
       </div>
-      <div className="w-2/3 relative flex items-center justify-center">
+      <div className="w-full sm:w-2/3 relative sm:flex sm:items-center sm:justify-center">
```
Comment (jessicamcinchak) [inline pages/about.tsx:67]: the diagram isn't actually showing up on mobile yet - it's a tricky scenario where the parent of the Next `Image` needs a position which is then interfering with the flex/flex-col up on L6. 

ideas/edits welcome please!

File: components/PageNavbar.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ export const PageNavbar = (props: { variant?: "light" }) => {
   const { variant } = props
 
   return (
-    <nav className={`flex justify-between ml-8 mr-8 mt-3 h-12 ${variant == "light" ? "text-black" : ""}`}>
+    <nav className={`flex justify-between ml-4 sm:ml-8 mr-4 sm:mr-8 mt-3 h-12 ${variant == "light" ? "text-black" : ""}`}>
```
Comment (jessicamcinchak) [inline components/PageNavbar.tsx:8]: also very open to opinions about if the first screen breakpoint we want to target is actually `sm` or `md` ! 

tailwind references for this here: https://tailwindcss.com/docs/screens

File: pages/about.tsx (hunk 1 above)
Comment (DafyddLlyr) [reply pages/about.tsx:67]: Yeah I also hit this issue, where a Next `Image` is always wrapped in it's own div, so doesn't behave as you'd expect.

We could try to import `next/future/image` which resolves this (and is how images are handled in v13 onwards).

Happy to have a look at this and try to see what I can come up with as is though!

File: components/PageNavbar.tsx (hunk 2 above)
Comment (DafyddLlyr) [reply components/PageNavbar.tsx:8]: I think that `sm` would be mobile so what we're really aiming for here (at least as a priority) 👍 Thanks for the link to the docs thought 👍

**Review by DafyddLlyr — APPROVED** [review]
Looking great 💅 

Realised that me merging my `ContentPage` stuff has broken this though - sorry, should have checked first! 

If the rebase / conflicts is giving you any grief, please let me know and I'll take a look - hopefully it's just a case of applying these styles to the `ContentPage` component 👍

File: pages/about.tsx (hunk 1 above)
Comment (jessicamcinchak) [reply pages/about.tsx]: thanks for this tip - swapped our import for `next/future/image` and honestly just got way better console warnings/suggestions :sweat_smile: 

diagram margins aren't quite as clean/impactful on mobile yet but i think it'll do for mvp!
Screenshot from 2023-01-26 22-02-26

**Pull request merged**